pkg/component/component.go

```diff
@@ -40,14 +40,7 @@ func GetLabels(componentName string, applicationName string, additional bool) ma
 	return labels
 }
 
-func CreateFromGit(client *occlient.Client, name string, ctype string, url string) (string, error) {
-	// if current application doesn't exist, create it
-	// this can happen when ocdev is started form clean state
-	// and default application is used (first command run is ocdev create)
-	currentApplication, err := application.GetCurrentOrDefault(client)
-	if err != nil {
-		return "", errors.Wrapf(err, "unable to create git component %s", name)
-	}
+func CreateFromGit(client *occlient.Client, currentApplication string, name string, ctype string, url string) (string, error) {
```

Comment:
the argument shouldn't be `currentApplication` but just `applicationName`. 
As in the new context, it doesn't have to be a current application. 
(name can't be just `application` as we already have application package)